### PR TITLE
Add a guide for creating documentation images

### LINKED ISSUE #6173: Consider using an image format other than PNG
**Your Godot version:** N/A

**Issue description:**
At the moment, the doc writing guidelines say

> Before you add or replace any images in the documentation, they should be run through a PNG compressor to save size. You can use the lossless OxiPNG compressor included in [Squoosh](https://squoosh.app/) for this purpose. For heavier images, consider using a lossy compressor like [pngquant](https://pngquant.org/). With it, almost no image quality is lost during compression.

In my experience, lossless WebPs tend to be smaller than PNGs. [I wrote a Bash script that converts all of the images in this repo to lossless WebPs.](https://gist.github.com/Jayman2000/6dab562039cfc3bd44f37e2a46244ebe) On my system, all of the PNG files take up 49MiB of space. Once you convert them to lossless WebPs, they take up 36MiB of space. We could probably save even more space by making some of the images lossy WebPs or AVIF files.

I’m not sure if PDF or EPUB files support these newer image formats, so it might make sense to keep the source files as PNGs and then convert them to another format when building the HTML documentation.

**URL to the documentation page (if already existing):**
<https://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#image-contribution-guidelines>

### PR DESCRIPTION
**Pull request by skyace65**
Creates a dedicated page in the contributing section for adding documentation images. This page acts as a basic guide for taking screenshots, converting to webp if necessary, how to use Krita to crop or resize images, and adding images to the documentation. This page also writes out several standards:
- Screenshots should be created from a 1080p screen or scaled down
- Outlines and arrows should use the color yellow (multiple outlines are an exception but avoid red)
- Outline should be 2 pixels wide (it's what I've been doing)
- Text should be made with a simple easy to read font (obvious, but probably best to have it written down)

This also changes the standard for image format from webp to png given recent discussions in rocket chat. I know there was talk about doing lossy webp compression for images over a certain size, what do we want to make the cutoff?

Lastly, when I started making this PR I was going to add a full tutorial for creating outlines in Inkscape. That section quickly became huge. If people want I'm willing to write it but I'm not sure if we want to add something that large to the page especially when community tutorials exist.

Closes #6173

### CODE REVIEW

File: community/contributing/docs_image_guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
+.. _docs_image_guidelines:
+
+Creating Documentation Images
+=============================
+
+Throughout documentation images are often needed to make the explanation
+of a feature or concept as clear as possible for a reader. This page will
+explain the process from beginning to end
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:8]:
```suggestion
explain the process from beginning to end.
```

File: community/contributing/docs_image_guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+------------------
+
+To take a picture of something in Godot a screen capture tool can be used.
+On Windows 10 and 11 that would be the Snip & Sketch program. 
+
+On macOS pressing :kbd: `Shift + Command + 3`` lets you take a screenshot
+of a portion of the screen. After pressing those keys click and drag over
+the area you wish to take a picture of. To take a picture of the entire
+screen press :kbd: `Shift + Command + 4`. All screenshots taken will
+be saved to the desktop.
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:20]:
```suggestion
To take a picture of something in Godot a screen capture tool can be used.
On Windows 10 and 11 that would be the Snip & Sketch program.
Pressing :kbd:`Windows + Shift + S` lets you take a screenshot
of a portion of the screen and save it to the clipboard.
After pressing those keys, click and drag over
the area you wish to take a picture of.

On macOS, pressing :kbd:`Shift + Command + 3` does the same.
To take a picture of the entire screen press :kbd:`Shift + Command + 4`.
All screenshots taken will be saved to the desktop.
```

File: community/contributing/docs_image_guidelines.rst [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+All screenshots should ideally be taken on a 1080p screen. Anything higher
+resolution is adding detail that doesn't make the documentation better and
+dramatically increases file size. If you're taking screenshots on a higher
+resolution screen the screenshot should be scaled down. There are instructions
+on  how to do this later in this page.
+
+Format conversion
+-----------------
+
+The current format for images in Godot's documentation is .Webp. While some
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:36]:
```suggestion
The current format for images in Godot's documentation is WebP (.webp). While some
```

File: community/contributing/docs_image_guidelines.rst [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+linux programs will support saving screenshots in this format, macOS and the
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:37]:
```suggestion
Linux programs will support saving screenshots in this format, macOS and the
```

File: community/contributing/docs_image_guidelines.rst [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+Snip & Sketch program on windows do not. For images that don't need editing
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:38]:
```suggestion
Snip & Sketch program on Windows do not. For images that don't need editing
```

File: community/contributing/docs_image_guidelines.rst [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+done to them, such as precise cropping or adding outlines, Squoosh can be used.
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:39]:
```suggestion
Snip & Sketch program on Windows do not. For images that don't need editing,
such as precise cropping or adding outlines, Squoosh can be used.
```

File: community/contributing/docs_image_guidelines.rst [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+`Squoosh <https://squoosh.app/>`_ is a converter developed by google, is open
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:40]:
```suggestion
`Squoosh <https://squoosh.app/>`_ is a converter developed by Google, is open
```

File: community/contributing/docs_image_guidelines.rst [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+source, and doesn't give google any image rights by using it. There is also a
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:41]:
```suggestion
source, and doesn't give Google any image rights by using it. There is also a
```

File: community/contributing/docs_image_guidelines.rst [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+experimental CLI version which can be found `here <https://github.com/GoogleChromeLabs/squoosh/tree/dev/cli>`_`
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:42]: I think the link format needs to be fixed, and a '.' be added at the end of the sentence.

File: community/contributing/docs_image_guidelines.rst [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+But for most UI images some extra work should be done, specifically cropping to
+make an image look clean. Below is an example of good cropping.
+
+.. image:: img/cropped_image.webp
+
+For cropping Krita is the recommended program. While some screenshot programs do
+have cropping built in it's not always easy to get something precise. And while
+Krita is designed as a painting program the cropping tool gives you pixel precision
+by default. However if there's a different program you use that fits this workflow
+well feel free to use it.
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:57]:
```suggestion
by default. Of course, feel free to use a different program you are familiar with.
```

File: community/contributing/docs_image_guidelines.rst [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+As explained earlier on this page, all images taken on a screen that is a higher resolution
+than 1080p should be scaled down. To do this in Krita click on **Image** on the top bar, and
+from the dropdown menu select **Scale Image To New Size**. This menu can also be opened by
+pressing :kbd: `ctrl + alt + I`. On this menu you want to adjust the pixel dimensions. For
+anything taken on a 4K monitor change the value of the width and height to half of its current
+value, for anything taken on a 1440p monitor multiply the width and height by .75. Make
+sure the **Constrain Proportions** box at the bottom of the menu is checked so you only have
+to change 1 value.
+
+Saving as webp in Krita
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:92]:
```suggestion
Saving as WebP in Krita
```

File: community/contributing/docs_image_guidelines.rst [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+-----------------------
+
+To save an image as webp if it isn't already one, Go to **file > save as**. Select **webp** from the
```
Comment (mhilbrunner) [inline community/contributing/docs_image_guidelines.rst:95]: `file > save as` probably is capitalized?

**Review by mhilbrunner — COMMENTED** [review]
Thanks for working on this! Left some comments/suggestions.

**Comment by skyace65** [comment]
Fixed.

**Comment by skyace65** [comment]
Superseded by #6627

**Pull request closed without merging**